packages/api-explorer/src/lib/upgrade-legacy-responses.js

```diff
@@ -13,6 +13,11 @@ module.exports = responses => {
   const codes = {};
 
   responses.forEach(response => {
+    // If there's no status set, then the legacy response wasn't full set up so we should ignore it.
```

Comment:
these comments are great; minor sp. suggestion

```suggestion
    // If there's no status set, then the legacy response wasn't fully configured so we should ignore it.
```